modules/protobuf/29.0-rc1/presubmit.yml

```diff
@@ -0,0 +1,36 @@
+matrix:
+  platform: ["debian10", "macos", "ubuntu2004", "windows"]
+  bazel: [6.x, 7.x]
```

Comment:
6.x is no longer supported in Bzlmod due to `rules_jvm_external` using `use_repo_rule`.